gallery/src/components/NxGlobalSidebar/NxGlobalSidebarPage.tsx

```diff
@@ -0,0 +1,226 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import CodeExample from '../../CodeExample';
+import { NxP, NxCode, NxTable } from '@sonatype/react-shared-components';
+
+import './NxGlobalSidebarPage.scss';
+
+const NxGlobalSidebarExample = require('./NxGlobalSidebarExample.tsx?raw'),
+    NxGlobalSidebarWithPageSidebarExample = require('./NxGlobalSidebarWithPageSidebarExample.tsx?raw');
+
+export default function NxGlobalSidebarPage() {
+  return (
+    <>
+      <section className="nx-tile">
```

Comment:
Is there a reason this isn't using `GalleryDescriptionTile`?

Reply:
> Is there a reason this isn't using `GalleryDescriptionTile`?

No reason that I can remember: https://github.com/sonatype/sonatype-react-shared-components/pull/371/commits/684ad398f518d2d3bfc1bbc3cc36ebf2d3abefd4